static/src/javascripts/projects/common/modules/commercial/dfp/dfp-api.js

```diff
@@ -783,7 +781,7 @@ define([
             rendered = false;
             adverts = {};
             slotsToRefresh = [];
-            mediator.off('window:resize', windowResize);
+            mediator.removeAllListeners('window:resize');
```

Comment:
Yeah, I did see this as a bit of a kludge. Think I've now found something that both we and the linter are happy with, though.